Update laboratory1.sql

larionovajul opened the pull request.

larionovajul committed:
Update laboratory1.sql

Kolobayeva commented on km52/Larionova_Yuliia/laboratory1.sql:
```diff
@@ -1,2 +1,135 @@
+
+---------------------------------------------------------------------------*/
+--Код відповідь:
+create user larionova identified by pass
```
use capital letters

larionovajul replied:
It's not necessary  to use capital letters in SQL.

Kolobayeva commented on km52/Larionova_Yuliia/laboratory1.sql:
```diff
@@ -1,2 +1,135 @@
+
+
+create table desks(
+    desk_name varchar2(40) not null
```
`desk_material varchar2(40) not null,`
`desk_description varchar2(40) not null,`
`desk_age number(10, 0) not null`

Kolobayeva commented on km52/Larionova_Yuliia/laboratory1.sql:
```diff
@@ -1,2 +1,135 @@
+GRANT CREATE ANY TABLE TO larionova ;
+GRANT ALTER ANY TABLE TO larionova ;
+GRANT SELECT ANY TABLE TO larionova ;
+GRANT UPDATE ANY TABLE TO larionova;
```
~~`GRANT UPDATE ANY TABLE TO larionova;`~~

Kolobayeva replied:
`GRANT INSERT ANY TABLE TO larionova;`

larionovajul replied:
> `GRANT INSERT ANY TABLE TO larionova;`

It's not necessary since  I already granted insert in the first task.
https://github.com/LarionovaJulia/dbis/blob/39c64cc8228f89485692e1d7fdba3689d6a6f578/km52/Larionova_Yuliia/laboratory1.sql#L18

Kolobayeva commented on km52/Larionova_Yuliia/laboratory1.sql:
```diff
@@ -1,2 +1,135 @@
+---------------------------------------------------------------------------*/
+
+--Код відповідь:
+select distinct count(cust_email) count_email
```
`select distinct count(cust_email) as "count_email"`

larionovajul replied:
It's not necessary to use quotes here. Quotes are only needed if you use reserved key words like `select`, `from`, `table`, etc.

larionovajul committed:
Update laboratory1.sql

larionovajul committed:
Homework

Add insert table and check constraint for chairs, desks, classrooms, buildings tables. Create new table - buildings.

larionovajul committed:
Add ERD

larionovajul committed:
Update laboratory2.sql

larionovajul committed:
Update laboratory3.sql

larionovajul committed:
Visualisation with python

larionovajul committed:
Merge branch 'master' of https://github.com/LarionovaJulia/dbis

larionovajul committed:
Update laboratory5.sql

larionovajul committed:
Importing from csv

larionovajul committed:
Update laboratory4.sql
